core/db/migrate/20151001121454_create_spree_shipping_methods_stock_locations.rb

```diff
@@ -0,0 +1,8 @@
+class CreateSpreeShippingMethodsStockLocations < ActiveRecord::Migration
+  def change
+    create_table :spree_shipping_methods_stock_locations do |t|
+      t.belongs_to :shipping_method
+      t.belongs_to :stock_location
```

Comment:
Can you add foreign keys for referential integrity? Its available in rails now, so we might as well add it in. I also feel like these should have `index: true` on them.